modules/widgets/authors.php

```diff
@@ -90,14 +90,27 @@ public function widget( $args, $instance ) {
 		 */
 		$excluded_authors = apply_filters( 'jetpack_widget_authors_exclude', $default_excluded_authors );
 
-		$authors = get_users(
+		/**
+		 * Filter the parameters of `get_users` call in the Widget Authors widget.
+		 *
+		 * See the following for `get_users` default arguments:
+		 * https://codex.wordpress.org/Function_Reference/get_users
+		 *
+		 * @module widgets
+		 *
+		 * @since 7.6.0
+		 *
+		 * @param array Array of params used in `get_user`
+		 */
+		$get_author_params = apply_filters( 'jetpack_widget_authors_params',
```

Comment:
This will make phpcs happy :)

```suggestion
		$get_author_params = apply_filters(
			'jetpack_widget_authors_params',
```